wallet-new/src/Cardano/Wallet/API/V1/Swagger.hs

```diff
@@ -229,14 +233,45 @@ Error Name / Description | HTTP Error code | Example
 $errors
 |] where
   errors = T.intercalate "\n" rows
-  rows = map (mkRow errToDescription) Errors.sample
+  rows =
+    [ mkRow errToDescription $ NotEnoughMoney 1400
+    , mkRow errToDescription $ OutputIsRedeem sampleAddress
+    , mkRow errToDescription $ MigrationFailed "Migration failed."
+    , mkRow errToDescription $ JSONValidationFailed "Expected String, found Null."
+    , mkRow errToDescription $ UnknownError "Unknown error."
+    , mkRow errToDescription $ InvalidAddressFormat "Invalid Base58 representation."
+    , mkRow errToDescription WalletNotFound
+    , mkRow errToDescription $ WalletAlreadyExists sampleWalletId
+    , mkRow errToDescription AddressNotFound
+    , mkRow errToDescription $ MissingRequiredParams (("wallet_id", "walletId") :| [])
+    , mkRow errToDescription $ WalletIsNotReadyToProcessPayments sampleSyncProgress
+    , mkRow errToDescription $ NodeIsStillSyncing (mkSyncPercentage 42)
```

Comment:
Meh. This lacks some of the available errors. Would be nice to have a generic way to generate those, or if not, something that will crash at runtime upon starting if one of them is missing.

Reply:
TODO